core/core.go

```diff
@@ -92,25 +92,24 @@ func (query Query) String() string {
 }
 
 var spaces = regexp.MustCompile(`\s+`) //nolint:gochecknoglobals
+var tagClean = regexp.MustCompile(`(^#|#$)`)
+var tagFetch = regexp.MustCompile(`#\w+([^#]*\w#)?`)
 
 func ParseQuery(arg string) Query {
 	query := Query{Tokens: spaces.Split(norm.NFC.String(arg), -1)}
 
-	query.Tags = make([]string, 0, len(query.Tokens))
-	words := make([]string, 0, len(query.Tokens))
+	queryArgString := norm.NFC.String(arg)
+	query.Tags = []string{}
```

Comment:
This line is unnecessary or can be replaced by `query.Tags = nil` if we expect it to have content for some reason. `append` will create a new slice from the contents of the previous slice, creating a new slice if it is `nil`.